azext_edge/edge/params.py

```diff
@@ -100,6 +100,15 @@ def load_iotops_arguments(self, _):
             arg_type=get_three_state_flag(),
             help="Include mq traces in the support bundle. Usage may add considerable size to the produced bundle.",
         )
+        context.argument(
+            "include_arc_agents",
+            options_list=["--arc"],
+            arg_type=get_three_state_flag(),
+            help="Include ARC agents resources in the support bundle. Note that logs from evicted "
```

Comment:
this applies for arc pods only correct?

Reply:
the log part? it will be for all the pod resource, not only arc according to the discussion we had in the meeting

Reply:
if it is for all - then why have the param description describe the behavior here? it implies that this will only apply to arc pods

Reply:
true true, I will update the help